docs/src/components/orb-core/README.md

```diff
@@ -0,0 +1,60 @@
+# Orb Core
+
+`orb-core` contains the core rust application responsible for verifying users'
+World IDs.
+
+The binaries controlling the orb are found in src/bin/:
+
+- src/bin/orb-core.rs: the production binary, which runs verifications in the field;
+- src/bin/orb-backend-connect.rs: a binary to ensure backend connectivity by scanning a WiFi QR code and establishing the WiFi connection as long as the backend is not reachable
+
+## Code Overview
```

Comment:
This flowchart is for do_signup, but the current docs imply it's all of orb core. Lots more going on besides this.